src/components/_global/BalLazy/BalLazy.vue

```diff
@@ -0,0 +1,11 @@
+<script setup lang="ts">
```

Comment:
taken from here https://medium.com/js-dojo/lazy-rendering-in-vue-to-improve-performance-dcccd445d5f